src/topp/QualityControl.cpp

```diff
@@ -143,6 +145,8 @@ class TOPPQualityControl : public TOPPBase
     setValidFormats_("in_trafo", {"trafoXML"});
     registerTOPPSubsection_("MS2_id_rate", "MS2 ID Rate settings");
     registerFlag_("MS2_id_rate:assume_all_target", "Forces the metric to run even if target/decoy annotation is missing (accepts all pep_ids as target hits).", false);
+    registerStringOption_("out_evd", "<Path>", "/buffer/ag_bsc/pmsb_2021/musch/mq_out", "EvidenceTXT with QC information", false); // TODO: better description
```

Comment:
remove this default; set it to `""` (you can set the path in your TOPPAS pipeline)